helsingborg-stad/app-mitt-helsingborg
# Feat/summary card component

## Explain the changes you’ve made

I have updated the CaseCard component to be a typescript file, and written tests to confirm its functionality. I have also updated the CaseCard component to be able to display a date and time in order to be used in a summary card for a booking.

## Explain why these changes are made

The additions (primarily a box that shows a date and time) are necessary for showing the user that they have successfully booked a meeting. 

## Explain your solution

The component was updated by rewriting it as a typescript file, and adding new props that conditionally render a box with time and date.

## How to test the changes?

1. Checkout this branch
2. Run `yarn install` to install the new packages (`npm install` will not work correctly due to a bug with Jest 27 that requires manual resolution as a workaround until further notice. `npm` cannot do this but `yarn` can).
3. Run the tests by running `npm run test`
4. Enable storybook mode in the `.env` file
5. Run the application in the simulator by running `npm run ios`
6. The new component can be found at "Booking examples" under "Case card" in the storybook navigator

## Was this feature tested in the following environments?
- [] Building the Application on a Android device/simulator.
